### PR TITLE
Bugfix/apps 260 hard delete

### PR DESCRIPTION
**Pull request by jokedurnez**
```
java.lang.IllegalArgumentException: Service methods cannot return void.
```

Also the default ports clash with other services.

### CODE REVIEW

File: src/test/java/com/openlattice/chronicle/ChronicleTestUtils.java [hunk 1]
```diff
@@ -99,7 +99,7 @@ public static OkHttpClient createHttpClient( Interceptor... interceptors ) {
 
     public static Retrofit createRetrofitAdapter( OkHttpClient httpClient ) {
         return RetrofitBuilders.decorateWithRhizomeFactories( RetrofitBuilders
-                .createBaseChronicleRetrofitBuilder( "http://localhost:8081/rhizome/api/", httpClient ) )
+                .createBaseChronicleRetrofitBuilder( "http://localhost:8090/rhizome/api/", httpClient ) )
```
Comment (katherinebernstein) [inline src/test/java/com/openlattice/chronicle/ChronicleTestUtils.java:102]: what's the reason for this change?

Comment (jokedurnez) [reply src/test/java/com/openlattice/chronicle/ChronicleTestUtils.java:102]: it clashes with indexer I believe?

File: src/main/java/com/openlattice/chronicle/ChronicleStudyApi.java [hunk 2]
```diff
@@ -110,7 +110,7 @@ void deleteParticipantAndAllNeighbors(
      * @param studyId - studyId
      */
     @DELETE( BASE + STUDY_ID_PATH )
-    void deleteStudyAndAllNeighbors(
+    Void deleteStudyAndAllNeighbors(
```
Comment (makosblade) [inline src/main/java/com/openlattice/chronicle/ChronicleStudyApi.java:113]: So I think the recommended thing to do according to Retrofit is to use a `Response` object.

**Comment by jokedurnez** [comment]
Upon a reading up, it looks like Retrofit2 wants a response, and the best way would be to use `Void`.  Whatever we replace it with, `void` is impossible.
https://stackoverflow.com/questions/33228126/how-can-i-handle-empty-response-body-with-retrofit-2

What is curious is that the API works, but I can't get the tests in rehearsal working with `void`.  They come with that error in the PR description.   I've looked through the codebase, any Void/void function that is tested is Void.  All void-functions are not tested.
Since we have a bunch of `Void`-api-calls in the codebase, I would suggest we keep `Void`, and if we ever want to change, we can change all API's at once.
